test/dest.js

```diff
@@ -763,53 +763,6 @@ describe('dest stream', function() {
     stream.end();
   });
 
-  it('should write file atime and mtime using the vinyl stat', function(done) {
```

Comment:
@erikkemperman I'm going to back out the vinyl changes and reopen your PR there.